coremltools/converters/mil/frontend/torch/internal_graph.py

```diff
@@ -439,12 +441,23 @@ def from_exir(cls, exir):
 
             params[name if name not in parameters_to_inputs else parameters_to_inputs[name]] = value
 
-        graph = exported_program.graph
+        graph_module = exported_program.graph_module
+        graph = graph_module.graph
 
         outputs = []
-        for node in graph.nodes:
+        for node_without_typing in graph.nodes:
+            node: torch.fx.Node = node_without_typing
```

Comment:
why do we iterate through the name of `node_without_typing`, and then immediately assign the value to another variable called `node`, and not using `node_without_typing` no more?
This might will be a little bit confusing ...
If you want to emphasize the fact the node doesn't have typing,
either replace all `node` in the for loop with `node_without_typing`,
or keeping the name `node` but add a comment in the beginning of the for loop, explaining the node doesn't have typing,
might will be more clear?

Reply:
Ah this is to let typing hint work: explicitly assign typing to `node`

I agree this makes the code itself confusing. Removed line 449 and use `node` only